src/components/numberInput/index.tsx

```diff
@@ -47,6 +47,7 @@ export type NumberInputProps = React.PropsWithRef<
 };
 
 function NumberInput(props: NumberInputProps, ref: any) {
+  const themeProps = useThemeProps(props, 'NumberInput');
```

Comment:
Didn't you add generics to useThemeProps so you can send it here?

Reply:
I did, but it works without it 🪄 
Pretty sure that it gets the type from the input `props` and that's why I got the errors in `Picker` and `Avatar`